src/Raven.Server/Documents/Queries/Results/GraphQueryResultRetriever.cs

```diff
@@ -64,9 +62,12 @@ public override bool TryGetKeyCorax(IndexSearcher searcher, long id, out Unmanag
             throw new NotSupportedException("Graph Queries do not deal with Corax indexes.");
         }
 
-        public override Document DirectGet(ref RetrieverInput retrieverInput, string id, DocumentFields fields) => DocumentsStorage.Get(_context, id);
-
-        protected override Document LoadDocument(string id) => DocumentsStorage.Get(_context, id);
+        protected override Document LoadDocument(Document parent, string id, ref RetrieverInput retrieverInput)
+        {
+            return id == string.Empty 
```

Comment:
Is it valid to pass empty string as ID to LoadDocument?

Reply:
Yes, it is:
https://github.com/ravendb/ravendb/pull/18132#discussion_r1531651071